src/jit/emitxarch.cpp

```diff
@@ -3982,6 +3982,14 @@ void emitter::emitIns_R_A(instruction ins, emitAttr attr, regNumber reg1, GenTre
     emitHandleMemOp(indir, id, IF_RRW_ARD, ins);
 
     UNATIVE_OFFSET sz = emitInsSizeAM(id, insCodeRM(ins));
+    if (ins == INS_crc32)
```

Comment:
nit: It would be nice if we could update the more central logic (either in `emitInsSize*` or one of the common methods they share).

Reply:
It would be nice. Perhaps add an `emitAdjustSizeCrc32()` and call it for these. Not ideal, but one doesn't want to add additional checks on other paths.